### PR TITLE
androiddev, django, express, angular

### PR DESCRIPTION
**Pull request by rosetanya898**

### CODE REVIEW

File: content/angular/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- Testing.
+
+
+
+## RESOURCES 😎 
+
+
+**Online Courses** 💻
+
+- [Angular - The Complete Guide (2021 Edition)](https://www.udemy.com/)
```
Comment (irenekurien) [inline content/angular/index.md]: Please add the correct link to udemy.

File: content/django-path/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+26.	Deployment
+
+
+## RESOURCES 😎 
+
+
+**Online Courses** 💻
+
+- [Django for Everybody Specialization](https://www.coursera.org/specializations/django)
+- [Python Django - The Practical Guide](https://www.udemy.com/)
```
Comment (irenekurien) [inline content/django-path/index.md]: Please check and add the correct link

File: content/dsa/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- Searching and Sorting Techniques
+
+
+
+## RESOURCES 😎 
+
+
+**Online Courses** 💻
+
+- [Udemy](https://www.udemy.com/)
```
Comment (irenekurien) [inline content/dsa/index.md]: Please check the link here also.

**Review by irenekurien — CHANGES_REQUESTED** [review]

**Review by rosetanya898 — COMMENTED** [review]
Modified links

**Pull request merged**